### PR TITLE
List the new bot rank in /groups

### PR DESCRIPTION
**Pull request by panpawn**

### CODE REVIEW

File: chat-plugins/info.js [hunk 1]
```diff
@@ -922,6 +922,7 @@ exports.commands = {
 			"+ <b>Voice</b> - They can use ! commands like !groups, and talk during moderated chat<br />" +
 			"% <b>Driver</b> - The above, and they can mute. Global % can also lock users and check for alts<br />" +
 			"@ <b>Moderator</b> - The above, and they can ban users<br />" +
+			"* <b>Bot</b> - The above, and they can also declare<br />" +
```
Comment (Zarel) [inline chat-plugins/info.js]: on second thought, I don't think we necessarily want it here

Comment (Zarel) [inline chat-plugins/info.js]: Maybe just "Like Moderator, but makes it clear that this user is a bot"

**Comment by panpawn** [comment]
Should it say that it has declare permissions?

**Comment by Zarel** [comment]
that's probably not relevant enough for this

**Comment by Zarel** [comment]
I don't think bots even have declare permissions in Main...

**Comment by panpawn** [comment]
They do

**Pull request merged**